crates/swc/src/plugin.rs

```diff
@@ -152,7 +164,7 @@ impl RustPlugins {
 
                 // Plugin transformation is done. Deserialize transformed bytes back
                 // into Program
-                serialized.deserialize()
+                serialized.deserialize().map(|mut v| v.take())
```

Comment:
Why do we need  `.take()` in `map`?

Reply:
I checked it and I think we can add `into_inner()` to `VersionedSerializable`

Reply:
That sounds legit to me.

Reply:
Actually, I misread this: `serialized.deserialize()` returns `Result<VersionedSerializable<Program>, Error>` and we need to return `Result<Program, Error>` by taking out VersionedSerializable, so here we call `Result::map`. Implementing `into_inner`  to `VersionedSerializable` can resolve this without `map`?

Reply:
If it's a result, `map` is fine. I thought it was an option